db/fixtures/miq_user_roles.yml

```
@@ -36,6 +36,9 @@
   - container_dashboard
   - ems_container
   - configuration_job
+  - configuration_manager
```

I'm pretty sure the remaining 2 are not needed and this should be:
```suggestion
  - configuration_management
```

This way the whole RBAC hierarchy is underneath it and will behave as before the change.